src/base58.h

```diff
@@ -146,7 +146,8 @@ template<typename K, int Size, CChainParams::Base58Type Type> class CBitcoinExtK
 
     K GetKey() {
         K ret;
-        ret.Decode(&vchData[0], &vchData[Size]);
+		if (vchData.size() == 74) //if base58 encouded data not holds a ext key, return a !IsValid() key
```

Comment:
OK I understand now why. ret.Decode takes a fixed-size object of 74 bytes.
In that case, please define a constant.